{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -77,7 +77,7 @@ repos:
       done"
 
 - repo: https://github.com/anmut-consulting/pipenv-setup
-  rev: v3.1.5
+  rev: v3.1.6
```

Comment:
WARNING: I've updated the version of our pipenv-setup hook in anticipation of merging into master [this PR](https://github.com/anmut-consulting/pipenv-setup/pull/7).